README.md

```diff
@@ -1,7 +1,7 @@
 # Bot Builder SDK v4
  | Branch | Description        | Build Status | Coverage Status |
  |----|---------------|--------------|-----------------|
-|Master | 4.2.* Preview Builds |[![Build status](https://travis-ci.org/Microsoft/botbuilder-js.svg?branch=master)](https://travis-ci.org/Microsoft/botbuilder-js) |[![Coverage Status](https://coveralls.io/repos/github/Microsoft/botbuilder-js/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/botbuilder-js?branch=master)
+|Master | 4.2.* Preview Builds |[![Build status](https://fuselabs.visualstudio.com/SDK_v4/_apis/build/status/BotBuilder-JS%204.0%20master)](https://fuselabs.visualstudio.com/SDK_v4/_build/latest?definitionId=374)|[![Coverage Status](https://coveralls.io/repos/github/Microsoft/botbuilder-js/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/botbuilder-js?branch=master)
```

Comment:
I don't know if this is right. I think devops also needs the "branch=master" URL paramter. 

W/o that, any time the build fails on any branch, the master branch would be marked as failed.

Reply:
"?branchName=master" is the AzureDevops paramter that's needed. 

You can see in the .Net version of this repo, where I did this for both Azure Devops and Coveralls:
Master | 4.3.* Preview Builds |[![Build status](https://fuselabs.visualstudio.com/SDK_v4/_apis/build/status/Bot.Builder-DotNet?branchName=master)](https://fuselabs.visualstudio.com/SDK_v4/_build/latest?definitionId=215&branchName=master) |[![Coverage Status](https://coveralls.io/repos/github/Microsoft/botbuilder-dotnet/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/botbuilder-dotnet?branch=master)

Reply:
Good save - I've added a commit to correct this, thanks!